Dockerfile

```diff
@@ -8,24 +8,25 @@
 # RUN:
 #   docker run -e VMPOOLER_CONFIG -p 80:4567 -it vmpooler
 
-FROM jruby:1.7-jdk
+FROM jruby:9.1-jdk
+
+RUN mkdir -p /var/lib/vmpooler && mkdir /var/log/vmpooler
 
-RUN mkdir -p /var/lib/vmpooler
 WORKDIR /var/lib/vmpooler
 
-ADD Gemfile* /var/lib/vmpooler
+ADD Gemfile* /var/lib/vmpooler/
```

Comment:
this * here failed for me when I had a Gemfile + a Gemfile.lock
I don't know why there is a * wildcard here?